package.json

```
@@ -143,6 +116,20 @@
           "default": "",
           "description": "If set, redirects the logs to a file."
         },
+        "languageServerHaskell.hieVariant": {
```

Nitpick, should probably name this server variant or something to be agnostic
```suggestion
        "languageServerHaskell.serverVariant": {
```

---

@alanz
> @jneira it looks like you made the necessary updates. What else needs to happen?

Not much more, i hope you didn't mind than i commited directly to the branch.
With the changes the extension worked for me in windows, switching between the three sever options and with both `-d`  and `-l` set.

> I was considering passing the intended exe into the call to `isHieInstalled`, which is why I added the parameter.

ok, i'll try to implement the change later